# This is program 26
try:
    a = int(input("Enter your name: "))
    print(a + 4)
except:
    print("some unknown error occurred!")
# we write the whole code inside "try" and write the error handling message inside the "except" dialog box
# In this way we can prevent program execution error and handle it successfully
# This is called exception handling
